ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -172,6 +172,39 @@ def test_breakdown_with_filter(self):
         person_response = self.client.get("/" + response["result"][0]["persons"]["url"]).json()
         self.assertEqual(len(person_response["results"][0]["people"]), 1)
 
+    def test_insight_trends_compare(self):
```

Comment:
Will update with better util. Added this test because we didn't have compare coverage for the new url format